packages/hadron-auto-update-manager/package.json

```diff
@@ -20,6 +20,7 @@
     "webpack": "./index.js",
     "require": "./index.js"
   },
+  "types": "index.d.ts",
```

Comment:
With these index.d.ts files, how much more work would it have been to just make the files typescript rather than provide types for .js files? Or will that break something else that needs it to be .js?